### ISSUE TITLE
Segmentation fault(core dumped)

### ISSUE THREAD

**Issue by programerr01** [wilix-team/iohook#307]
```
load_input_helper [1918]: Using XFree86 keyboard layout.
load_input_helper [1920]: XKB support is required to accurately determine keyboard characters!
Segmentation fault (core dumped)

```
## Enviornment 
Os: ubuntu 20.04
Node: v14.16.0
npm : 6.14.11

**Comment by foresteam** [comment]
Have the same problem on:
Manjaro (kernel 5.11.6-1)
Node v14.16.0-1
Npm v7.6.3-1

**Comment by programerr01** [comment]
the issue can be solved by using ioHook version *0.6.6*
```npm install iohook@0.6.6```
